tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -626,7 +626,7 @@ public function updateLastShareByCurrentUser(?TableNode $body):void {
 	 * @throws Exception
 	 */
 	public function theUserUpdatesTheLastShareWith(?TableNode $body):void {
-		$this->updateLastShareByCurrentUser($body);
+		$this->response = $this->updateLastShareByCurrentUser($body);
```

Comment:
```suggestion
		$this->setResponse($this->updateLastShareByCurrentUser($body));
```
IMO, this setResponse was created just for this sort of steps